os/include/tinyara/signature.h

```diff
@@ -0,0 +1,51 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __INCLUDE_TINYARA_SIGNATURE_H
+#define __INCLUDE_TINYARA_SIGNATURE_H
+
+#include <tinyara/config.h>
+#include <stdint.h>
+
+#ifndef CONFIG_ARCH_BOARD_RTL8721CSM
+#error "Signing Verification is not supported on this board"
+#endif
+
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(CONFIG_ARCH_BOARD_RTL8721CSM)
```

Comment:
This file is common header, so I think it is not good to add board specific config.

Reply:
will consider it and update later.